charts/rstudio-connect/values.yaml

```diff
@@ -211,12 +211,37 @@ launcher:
   customRuntimeYaml: false
   # -- User definition of launcher.kubernetes.profiles.conf for job customization
   launcherKubernetesProfilesConf: {}
-  # -- Image definition for the RStudio Connect Content InitContainer
-  contentInitContainer:
+  # -- Whether to use launcher templates when launching sessions. Defaults to true
+  useTemplates: true
+  # -- Values to pass along to the RStudio Connect session templating process
+  templateValues:
+    service:
+      type: ClusterIP
+      annotations: {}
+      labels: {}
+    pod:
+      annotations: {}
```

Comment:
Since template validation is such a rough spot for us right now, I wonder if you want to add a note regarding the restrictions for k8s labels and annotations? Otherwise, the template / job submission will fail.

Labels: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/

Annotations: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/